data-access-layer/eventmesh/src/ApiServerClient.js

```diff
@@ -691,7 +692,7 @@ class ApiServerClient {
       data: data
     };
     return Promise.try(() => apiserver.api[CONST.APISERVER.CONFIG_MAP.API_VERSION]
-      .namespaces(CONST.APISERVER.DEFAULT_NAMESPACE)[CONST.APISERVER.CONFIG_MAP.RESOURCE_TYPE].post({
+      .namespaces(CONST.APISERVER.DEFAULT_NAMESPACE)[CONST.APISERVER.CONFIG_MAP.RESOURCE_TYPE].post({ // Currently only admin controller calls this to create configs in default NS. Should not be used in any other context.
```

Comment:
Same query as above

Reply:
same here, these utility functions are used in BOSH based deployment to create some configmaps which are used as flags for shceduled updates. Hence I didn't disturb these and kept them as is, added comment so that it is not used elsewhere.